assets/js/src/form_editor/components/form_settings/styles_settings_panel.jsx

```diff
@@ -22,6 +22,9 @@ const BasicSettingsPanel = ({ onToggle, isOpened }) => {
     []
   );
   const settingsRef = useRef(settings);
+  useEffect(() => {
+    settingsRef.current = settings;
+  });
```

Comment:
It would be better to make this effect dependent on `settings` so that it is not triggered every time.

Reply:
Done.